modules/cert-manager/cluster_issuer_route53.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: ${clusterissuer_name}
+spec:
+  acme:
+    email: ${email_address}
+    server: https://acme-staging-v02.api.letsencrypt.org/directory
```

Comment:
have an option to switch sandbox/prod letsencrypt for both R53 and gcloud dns issuers